cime_config/testdefs/testlist_clm.xml

```diff
@@ -2059,10 +2058,10 @@
     </machines>
     <options>
       <option name="wallclock">00:40:00</option>
-      <option name="comment"  >Run ARCTICGRIS for future transient case (do not run this expensive test with every tag, but include it in the less frequent ctsm_sci testing)"</option>
+      <option name="comment"  >Run ARCTICGRIS for 1850 case (do not run this expensive test with every tag, but include it in the less frequent ctsm_sci testing)"</option>
     </options>
   </test>
-  <test name="SMS_Ln9" grid="ne0CONUSne30x8_ne0CONUSne30x8_mt12" compset="I1850Clm60Sp" testmods="clm/clm60cam6LndTuningMode_2013Start">
+  <test name="SMS_Ln9" grid="ne0CONUSne30x8_ne0CONUSne30x8_mt12" compset="ISSP245Clm50BgcCrop" testmods="clm/clm50cam6LndTuningMode_2013Start">
```

Comment:
Change compset as above.

Reply:
`./create_test SMS_Ln9.ne0CONUSne30x8_ne0CONUSne30x8_mt12.IHistClm60Sp.izumi_intel.clm-clm60cam6LndTuningMode_2013Start`
in progress on izumi